.gitignore

```diff
@@ -1 +1,2 @@
 node_modules
```

Comment:
To make it easy for the whole team, we can use  a generic gitgnore file for react projects. check the link here.
this will cover all aspects of files that needs to be ignored.

[https://www.toptal.com/developers/gitignore/api/react,node](https://www.toptal.com/developers/gitignore/api/react,node)